src/ToDo/Business/ITaskService.cs

```diff
@@ -13,4 +13,13 @@ public interface ITaskService
 	ValueTask<IImmutableList<ToDoTask>> GetAsync(TaskList list, CancellationToken ct);
 
 	ValueTask<IImmutableList<ToDoTask>> GetAllAsync(string displayName = "", CancellationToken ct = default);
+
+	Task AddStepAsync(string baseTaskListId, string baseTaskId, CheckListItem checkListItem, CancellationToken ct);
+
+	ValueTask<IImmutableList<CheckListItem>> GetStepsAsync(string baseTaskListId, string baseTaskId, CancellationToken ct);
+
+	Task DeleteStepAsync(string baseTaskListId, string baseTaskId, string checklistItemId, CancellationToken ct);
+
+	Task UpdateStepAsync(string baseTaskListId, string baseTaskId, CheckListItem checkListItem, CancellationToken ct);
```

Comment:
nit: I feel like these shouldn't have "Async" as a suffix, unless there are non-async versions of these already. Not sure what the accepted convention is for the unoplatform org

Reply:
It's convention for unoplatform.